frontend/tests/playwright-files/files/index.html

```
@@ -0,0 +1,15 @@
+<html lang="en">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)